scm/scm.go

```diff
@@ -214,3 +216,8 @@ type Team struct {
 type Authorization struct {
 	Scopes []string
 }
+
+type RepositoryInvitationOptions struct {
```

Comment:
Would be nice to document what is the content of the two fields, cf. the other comment regarding the same.